vllm/config.py

```diff
@@ -1835,6 +1851,9 @@ def _get_and_verify_max_len(
                 raise ValueError(
                     f"{msg} To allow overriding this maximum, set "
                     "the env var VLLM_ALLOW_LONG_MAX_MODEL_LEN=1")
+
+    if bert_config:
```

Comment:
Can we put this at line 1821 like this?

```python
if bert_config and "max_seq_lenght" in bert_config:
    derived_max_model_len = bert_config["max_seq_length"]
```

Otherwise it will disable the user override logic.